scripts/resize-functions.js

```diff
@@ -0,0 +1,136 @@
+const binarySearch = dim => function bin(start, end) {
```

Comment:
Did you actually write this? If not, please cite your sources.

Reply:
Thanks for reminding me about the binary search.